### DISCUSSION TITLE
Has Anyone Had Any Luck With Syncing Data With Separate Clipboard App?

### DISCUSSION THREAD

**Discussion by andrewcincotta** [hluk/CopyQ#2190] [General]
While I value CopyQ's ability to sync clipboard's between various operating systems, I like the GUI of different Mac apps better (Alfred, Recast, Paste, etc). Has anyone attempted to sync CopyQ tab documents with separate clipboard manager apps?

**Comment by boyanmolina** [comment]
I sync between two Windows installation, one at work, one at home. Works perfectly. Not sure if that answers your question though...
